paasta_tools/api/views/instance.py

```diff
@@ -873,7 +878,9 @@ def bounce_status(request):
         return response
 
     try:
-        return pik.bounce_status(service, instance, settings)
+        return pik.bounce_status(
+            service, instance, settings, is_eks=instance_type == "eks"
```

Comment:
`is_eks=instance_type == "eks"`
Fancy! Was a bit confused in what this actually meant. 

Not a blocker but do we want to make it easier to understand by doing something like this instead?
```python
is_eks = True if istance_type == "eks" else False
return pik.bounce_status(service, instance, settings, is_eks)
```

or if we want to keep it a one liner maybe `is_eks=(instance_type == "eks")` ?

Reply:
I can parenthesize this :)